beancount_dkb/ec.py

```diff
@@ -134,4 +135,11 @@ def _read_meta(fd):
                                              data.EMPTY_SET, postings)
                         )
 
+                # Closing Balance
+                meta = data.new_metadata(file_.name, 0)
```

Comment:
Should the second argument be `0` here? We're using `enumerate` in the `for` loop before this. So either use `-1` here or `len(transactions)` (if that makes sense), or just use `index + ` in the `enumerate/for`.

Reply:
I think within beancount, these are actually meant to refer to line numbers in the raw input file? Which would make all of them wrong. I was hoping at some point we could refactor the entire parser to go line-by-line and keep track line numbers starting from the top, not just from the beginning of the transactions list.

Still, agreed that -1 makes more sense for now. Changed & pushed.